lib/desktop/session.rb

```diff
@@ -90,6 +90,7 @@ def initialize(uuid: nil, type: nil, name: nil)
         @state = :new
         @created_at = Time.now
         @name = name
+        @job_id = ENV['FLIGHT_JOB_ID']
```

Comment:
I did consider using an option instead. I decided against it as options are available to be edited by the user. I couldn't think of any use cases for the user to want to set that as an option, and I thought it may be confusing to have it show up in the help as something that is not recommended to be used